modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/podweak.dm

```diff
@@ -56,3 +56,6 @@
 	regenerate_organs(human, src, visual_only = TRUE)
 	human.update_body(TRUE)
 
+/obj/item/organ/external/pod_hair
+	mutantpart_key = "pod_hair"
+	mutantpart_info = list(MUTANT_INDEX_NAME = "Ivy", MUTANT_INDEX_COLOR_LIST = list("#ffffff", "#ffffff", "#ffffff"))
```

Comment:
This shouldn't have to be here.

Reply:
Pretty sure it should. See also the mushroom cap:
```
/obj/item/organ/external/cap
	name = "fungal cap"
	desc = "Real. No cap."
	icon_state = "random_fly_1"

	mutantpart_key = "caps"
	mutantpart_info = list(MUTANT_INDEX_NAME = "Round", MUTANT_INDEX_COLOR_LIST = list("#FFFFFF"))
```
Removing it caused it to break.

Reply:
I'm talking about the whole thing, it's there and duplicated in another file, it doesn't need to be here too. Look one file down, it's literally the same thing again.